frontend/hooks/useRefreshToken.ts

```diff
@@ -26,9 +33,15 @@ const useRefreshToken = () => {
       }
     );
 
+    const refreshToken = response.data.refreshToken;
+
+    if (typeof window !== 'undefined') {
+      localStorage.setItem('refreshToken', refreshToken);
+    }
+
     setTokens({
       accessToken: response.data.accessToken,
-      refreshToken: response.data.refreshToken,
+      refreshToken,
```

Comment:
Can you change this to refreshToken: refreshToken for consistency?

Reply:
Well.... ES6 has a nifty way to deal with that redundant information... :(